src/logger-coralogix.js

```diff
@@ -18,6 +18,23 @@ const {
 
 let coralogixLogger = null;
 
+// todo: sanitizing the secrets should be better handled in the logging framework.
+const sanitize = (params) => {
+  const sanitizedParams = { ...params };
```

Comment:
@kptdobe this is not a deep clone. i.e. your are destroying the `__ow_headers.authorization` here.

Reply:
I'll fix it in the other branch, thanks for checking.
I had the `ferrum#deepclone` before but it does not work if `__ow_logger` is pretty (cannot copy Logger object). Thus I had to add more magic. Like here https://github.com/adobe/helix-dispatch/blob/alex-playground/src/index.js#L21-L44